airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -195,8 +201,9 @@ public ConnectionRead webBackendUpdateConnection(WebBackendConnectionUpdate webB
 
       // wait for this to execute
       JobInfoRead resetJob = schedulerHandler.resetConnection(connectionId);
+      final JobInfoRead completedResetJob = waitForJobToComplete(resetJob.getJob().getId());
```

Comment:
this endpoint (updating a connection, still needs to block and until the reset job is complete, so we have to add that logic here)

Reply:
Why is that?

Reply:
i guess we could just not. if the reset fails then the destination will have a combination of data from different schemas.